Makefile

```diff
@@ -1,59 +1,43 @@
-SITE_PACKAGES := $(shell python -c "from distutils.sysconfig import get_python_lib; print(get_python_lib())")
-VIRTUAL_ENV_ERR = Python Virtual environment is not active. Run `virtualenv -p python3 .venv/py3 && source .venv/py3/bin/activate`
+.DEFAULT_GOAL := lint
 
-all: ansible/roles
-
-$(SITE_PACKAGES): requirements.txt
-ifndef VIRTUAL_ENV
-	$(error $(VIRTUAL_ENV_ERR))
-endif
-	pip install -r requirements.txt
-	touch $(SITE_PACKAGES)
-
-pip: $(SITE_PACKAGES)
-
-ansible/roles: pip
-	cd ansible && ansible-galaxy install -r requirements.yml -p roles
-	touch ansible/roles
-
-images: ansible/roles
+images:
 	packer build packer/epoch.json
 
-setup-infrastructure: ansible/roles check-deploy-env
```

Comment:
I see [ansible/roles moved to Dockerfile](https://github.com/aeternity/infrastructure/pull/68/files#diff-3254677a7917c6c01f55212f86c57fbfR22) so the Makefile now assumes more to be run insider Docker container. Ok.